src/PhpParser/NodeVisitor/AddLoopVarTypeToForeachNodeVisitor.php

```diff
@@ -39,7 +39,7 @@ public function leaveNode(Node $node): ?array
         }
 
         $docNop = new Nop();
-        $docNop->setDocComment(new Doc(sprintf('/** @var %s $%s */', '\\' . Loop::class, 'loop')));
+        $docNop->setDocComment(new Doc('$loop = new \\' . Loop::class . '();'));
```

Comment:
That `sprintf` was so nice and readable. 😍 
String concatenation add more punctuation characters.

Reply:
Oh, I found it cryptic with all the `%` and random alpha symbols there :sweat_smile:

Reply:
Thank you for the super useful answer.
No I know why this change was made.